packages/server/package.json

```diff
@@ -113,6 +113,7 @@
     "trash": "5.2.0",
     "tree-kill": "1.2.2",
     "ts-node": "8.5.4",
+    "tsconfig-paths": "3.10.1",
```

Comment:
why did this need move from a dev-dep?

Reply:
Trying to find where it failed - might have only been locally when I was writing the test case for the patch being applied, and it wasn't... and then I noticed that we do indeed use the package in a prod situation: https://github.com/cypress-io/cypress/blob/11e99fc35a4cf1fe43abed07cd3f7fb2441fa461/packages/server/lib/util/ts_node.js#L30-L37

Reply:
Nice catch then!